.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ['3.5', '3.8']
+        toxenv: ['django22']
+        include:
+          - python-version: "3.8"
+            toxenv: "quality"
+          - python-version: "3.8"
+            toxenv: "docs"
+          - python-version: "3.8"
+            toxenv: "pii_check"
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install pip
+      run: pip install -r requirements/pip.txt
+
+    - name: Install Dependencies
+      run: pip install -r requirements/travis.txt
+
+    - name: Run Tests
+      run: tox -e ${{ matrix.toxenv }}
```

Comment:
we can just specify `tox` here and define which tox environment to run in `TOXENV` environment variable, we won't need a separate step for each step like you've defined for quality and other environments and we should add quality and other checks in `toxenv` array to simplify the matrix.

Reply:
example here: https://github.com/edx/edx-cookiecutters/blob/f3dfafea478fc95a785b808fa85b3cde9e1e7e33/.github/workflows/ci.yml